.travis-deploy.sh

```diff
@@ -17,5 +20,20 @@ if [[ "${TRAVIS_PULL_REQUEST}" = "false" && "$TRAVIS_OS_NAME" = "linux" && $TASK
     rsync -avz --delete --exclude .git/ ../packagename/ ./
     git add -A
     git commit -m "Update rendered version to ""$TRAVIS_COMMIT"
+
+    # Add astropy_helpers manually at the version in the cookiecutter template
+    git submodule add https://github.com/astropy/astropy-helpers astropy_helpers || true
+    git submodule update --init
+    cd astropy_helpers
+    helpers_version=$(jq -r ".astropy_helpers_version" $base_dir/cookiecutter.json)
+    # parse the json with jq to get the helpers version
+    git checkout $helpers_version
+    cp ah_bootstrap.py ../
+    cp ez_setup.py ../
+    cd ..
+    git add astropy_helpers ah_bootstrap.py ez_setup.py
+    git status
+    # we might not have changes to commit
+    git commit -m "Update astropy_helpers to ""$helpers_version" || true
```

Comment:
why is git status needed here?

Reply:
we might be running into the limits of my bash here. If `git add` leads to no changes being commited `git commit` fails.